### ISSUE TITLE
Reshape function

### ISSUE THREAD

**Issue by pachpandepriti** [keras-team/keras#12218]
Can anyone guide me how can I use reshape function?
encoded (?, 4)
encod (?, 2)
normalize (?, 2)
Complex_symbols (?,)
decode(?,)
concat (?, 2)
deco (?, 4)
decod (?, 4)
I have size (None,) in 2 layers decode and concat where I need the size to be (None,1)
how should I use a Reshape function in this case? Or is there any other option

Please make sure that the boxes below are checked before you submit your issue.
If your issue is an **implementation question**, please ask your question on [StackOverflow](http://stackoverflow.com/questions/tagged/keras) or [on the Keras Slack channel](https://keras-slack-autojoin.herokuapp.com/) instead of opening a GitHub issue.

Thank you!

- [ ] Check that you are up-to-date with the master branch of Keras. You can update with:
`pip install git+git://github.com/keras-team/keras.git --upgrade --no-deps`

- [ ] Check that your version of TensorFlow is up-to-date. The installation instructions can be found [here](https://www.tensorflow.org/get_started/os_setup).

- [ ] Provide a link to a GitHub Gist of a Python script that can reproduce your issue (or just copy the script here if it is short).
